setup.cfg

```diff
@@ -8,14 +8,12 @@ packages =
 python_requires =
     >=3.9
 install_requires =
-    scikit-misc
-    scanpy>=1.8.1
+    torchmetrics==0.6.2
```

Comment:
Why were these dependencies added?

Reply:
`pytorch-lightning` relies on `torchmetrics.utilities.data.get_num_classes` for some utility function necessary for `scvi-tools`. However, `torchmetrics.utilities.data.get_num_classes` is missing for `torchmetrics>0.6.*`.

Reply:
For `protobuf`, I encountered this error
```
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
 ```
 . This error seems prevalent across different applications (an online search would get you some examples) since it probably relates to breaking changes in the newest `protobuf==4.21.1`. 

Hence I decided to pin `protobuf` at 3.20.1.

Reply:
Huh weird. Could be worth pinging the scvi-tools people about the `torchmetrics`  thing.